src/nixupdate/nix_outdated.py

```diff
@@ -86,15 +87,15 @@ def _generate_sbom(target_path, runtime=True, buildtime=False):
 
 def _run_repology_cli(sbompath):
     LOG.info("Running repology_cli")
-    prefix = "repology_"
-    suffix = ".csv"
-    with NamedTemporaryFile(delete=False, prefix=prefix, suffix=suffix) as f:
-        cmd = (
-            "repology_cli "
-            f"--sbom_cdx={sbompath} --repository=nix_unstable --out={f.name}"
-        )
-        exec_cmd(cmd.split())
-        return f.name
+    repology_cli = repology.repology_cli.Repology()
+    repo = "--repository nix_unstable"
+    status = f"--sbom_cdx {sbompath}"
+    str_args = f"{repo} {status} "
+    return repology_cli.query(
+        repology.repology_cli.getargs(str_args),
```

Comment:
assemble the list of args here

Reply:
Fixed in https://github.com/tiiuae/sbomnix/commit/ecf9b87d14ceb3b3668dc5b05746fc2b754d8195